src/item.cpp

```diff
@@ -739,6 +739,17 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
             }, false );
             info.push_back( iteminfo( "BASE", string_format( _( "Material: %s" ), material_list.c_str() ) ) );
         }
+        if( damage_bash() > 0 || damage_cut() > 0 ){
+            if( conductive () && !has_flag ( "CONDUCTIVE" ) ) { 
+                info.push_back( iteminfo( "BASE", string_format( _( "This weapon conducts electricity." ) ) ) );
```

Comment:
Could we use some <good>, <bad> tags to make these description more fancy?
Also asterisk.